docs/ops.html

```diff
@@ -3992,6 +3980,358 @@ <h5 class="anchor-heading"><a id="kafka_streams_cache_monitoring" class="anchor-
+      <td>The total number of record acknowledgements sent.</td>
+    </tr>
+    <tr>
+      <td>acknowledgements-error-rate</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=acknowledgements-error-rate,client-id=([-.\w]+)</td>
+      <td>The average number of record acknowledgements that resulted in errors per second.</td>
+    </tr>
+    <tr>
+      <td>acknowledgements-error-total</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=acknowledgements-error-total,client-id=([-.\w]+)</td>
+      <td>The total number of record acknowledgements that resulted in errors.</td>
+    </tr>
+    <tr>
+      <td>fetch-latency-avg</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=fetch-latency-avg,client-id=([-.\w]+)</td>
+      <td>The average time taken for a fetch request.</td>
+    </tr>
+    <tr>
+      <td>fetch-latency-max</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=fetch-latency-max,client-id=([-.\w]+)</td>
+      <td>The maximum time taken for any fetch request.</td>
+    </tr>
+    <tr>
+      <td>fetch-rate</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=fetch-rate,client-id=([-.\w]+)</td>
+      <td>The number of fetch requests per second.</td>
+    </tr>
+    <tr>
+      <td>fetch-total</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=fetch-total,client-id=([-.\w]+)</td>
+      <td>The total number of fetch requests.</td>
+    </tr>
+    <tr>
+      <td>fetch-throttle-time-avg</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=fetch-throttle-time-avg,client-id=([-.\w]+)</td>
+      <td>The average throttle time in milliseconds.</td>
+    </tr>
+    <tr>
+      <td>fetch-throttle-time-max</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=fetch-throttle-time-max,client-id=([-.\w]+)</td>
+      <td>The maximum throttle time in milliseconds.</td>
+    </tr>
+    <tr>
+      <td>bytes-consumed-rate</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=bytes-consumed-rate,client-id=([-.\w]+)</td>
+      <td>The average number of bytes consumed per second.</td>
+    </tr>
+    <tr>
+      <td>bytes-consumed-total</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=bytes-consumed-total,client-id=([-.\w]+)</td>
+      <td>The total number of bytes consumed.</td>
+    </tr>
+    <tr>
+      <td>records-consumed-rate</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=records-consumed-rate,client-id=([-.\w]+)</td>
+      <td>The average number of records consumed per second.</td>
+    </tr>
+    <tr>
+      <td>records-consumed-total</td>
+      <td>kafka.consumer:type=consumer-share-fetch-manager-metrics,name=records-consumed-total,client-id=([-.\w]+)</td>
+      <td>The total number of records consumed.</td>
+    </tr>
```

Comment:
These are not defined in KIP-932, do we need to update the KIP documentation for these as well?